universum/modules/launcher.py

```diff
@@ -308,6 +295,19 @@ def _handle_postponed_out(self) -> None:
             item[0](item[1])
         self._postponed_out = []
 
+    def _get_teamcity_build_tag(self) -> Optional[str]:
+        if self.configuration.is_conditional:
+            return None  # conditional steps always succeed, no sense to set a tag
+        tag: str = self.configuration.fail_tag if self._error else self.configuration.pass_tag
+        return tag  # can be also None if not set for current Configuration
+
+    def _assign_teamcity_build_tag(self, tag: str) -> None:
```

Comment:
Used wording from the TeamCity website: https://blog.jetbrains.com/teamcity/2010/03/tags-in-teamcity/